.gitignore

```diff
@@ -4,7 +4,6 @@ config.erl
 install.mk
 rel/*.config
 rel/dev*
-rel/snap*
```

Comment:
That hasn't changed from before, but it's easy enough to add if that's desirable